app/services/import_planned_disbursements.rb

```diff
@@ -58,6 +58,13 @@ def lookup_activity(roda_identifier)
     nil
   end
 
+  def convert_value(value, header:, roda_identifier:)
+    ConvertFinancialValue.new.convert(value)
+  rescue
```

Comment:
Do we want to rescue a specific error here?

Reply:
I'm not sure. Anything that causes an exception will cause the transaction to rollback, and will be printed to the shell where the user is running the importer. I'm happy to make this more user-friendly if we think it's worth the time investment -- I was trading off the fact I'm under a tight deadline, and this is code for a one-off task for devs to run, which will be thrown away immediately afterward.

Reply:
I've had to refactor this to take in new requirements, and the latest version is now rescuing a specific error to catch this. This class no longer converts the value itself, it lets `PlannedDisbursementHistory` take care of that, so we need to rescue the conversion error to detect this type of failure.